sonic-chassisd/scripts/chassisd

```diff
@@ -184,7 +188,7 @@ class ModuleUpdater(logger.Logger):
         self.chassis = chassis
```

Comment:
I think that instead of modifying the existing `ModuleUpdater` class, we should implement `SmartSwitchModuleUpdater(ModuleUpdater)`. `SmartSwitchModuleUpdater` should be derived from `ModuleUpdater` and overwrite the methods that should behave differently for the Smart Switch. This approach allows us to keep the original implementation untouched and guarantees full backward compatibility with the chassisd.

Reply:
Will consider this

Reply:
Implemented this change.

Reply:
@oleksandrivantsiv

Reply:
#closed